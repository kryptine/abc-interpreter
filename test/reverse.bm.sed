s/1000/30000/
