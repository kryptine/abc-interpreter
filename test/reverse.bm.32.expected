30000
halt at 372
184083 340205 524288
